packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -668,263 +668,77 @@ contract OPContractsManagerUpgrader is OPContractsManagerBase {
         // Loop through each chain and upgrade.
         for (uint256 i = 0; i < _opChainConfigs.length; i++) {
             assertValidOpChainConfig(_opChainConfigs[i]);
+            uint256 l2ChainId = _opChainConfigs[i].systemConfigProxy.l2ChainId();
```

Comment:
why does this var (and the `Upgraded` event at the end of the loop need to happen outside of `_doChainUpgrade()`?